### ISSUE TITLE
Save user into db label:question 

### ISSUE THREAD

**Issue by Rickyc81** [aedoardo/telegram-twitch-sub-bot#4]
Hi, can you explane me what you do when call this url with telegram id in your backend?
```
url_message = "https://twitcham.com/telegram?tgid=" + str(jwt_code)  # build the url
```
When i click on "verify twitch account" it redirect to this link, probably open the Oauth2 link for Auth the account.. 
and then? you save on db the entire user profile and your telegram id? and save other stuff?

Thanks so much

Ric

**Comment by aedoardo** [comment]
Hi @Rickyc81 , it was not an OAuth2 but it was a simple page where I store the telegram id, the twitch id and the twitch username. If I remember correctly, on telegram?tgid= page I store the jwt into a cookie, then I redirect to the Twitch page in order to gain the info (twitch id and twitch username). Then, when the user allows to share his infos, he is redirected to a successful page where I put the data into the database (and I check also the jwt cookie).

**Comment by Rickyc81** [comment]
ok thanks.. as I thought ..

> then I redirect to the Twitch page in order to gain the info (twitch id and twitch username). Then, when the user allows to share his infos, he is redirected to a successful page where I put the data into the database (and I check also the jwt cookie)

so there is always an OAuth request for obtain twitch user profile

The user profile Schema is that in homepage?

**Comment by aedoardo** [comment]
Yes, it was the user profile schema (it is not very good). Yeah, the OAuth to Twitch :P
